ros2bag/ros2bag/verb/play.py

```diff
@@ -99,29 +95,29 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--playback-duration', type=float, default=-1.0,
             help='Playback duration, in seconds. Negative durations mark an infinite playback. '
-                 'Default is -1.0. When positive, the maximum of `playback-until-*` and the one '
-                 'that this attribute yields will be used to determine which one stops playback '
-                 'execution.')
+                 'Default is %(default)d. '
```

Comment:
not sure if we enforce a single string interpolation style in ROS 2, but I always assume f-strings are the way to go.

Reply:
FYI, one thing to be careful of is RHEL-8 support.  RHEL-8 only has Python 3.6, and we've had some problems with certain features because it is so old.  Not sure if that is the case here, just pointing it out.

Reply:
f-strings are good for many things! But they don't encode the builtin info that this argparse special token `%(default)` does. This way we only specify `default=` in the parameter description and the helpstring can use that same value, instead of duplicating it.

I'll double check whether it's supported in 3.6

Reply:
Update:  the argparse help tokens do work in python3.6.

However, the signature we are using for `importlib.metadata.entry_points` is not. @clalancette is there any CI job I can run to make sure I don't break RHEL-8?

Reply:
> @clalancette is there any CI job I can run to make sure I don't break RHEL-8?

Yep!  https://ci.ros2.org/job/ci_linux-rhel/